BraveRewardsUI/Wallet/WalletViewController.swift

```diff
@@ -159,7 +160,7 @@ class WalletViewController: UIViewController, RewardsSummaryProtocol {
     publisherSummaryView.monthlyTipView.addTarget(self, action: #selector(tappedMonthlyTip), for: .touchUpInside)
     
     publisherSummaryView.monthlyTipView.isHidden = true
-    publisherSummaryView.monthlyTipView.batValueView.amountLabel.text = "5"
+    publisherSummaryView.monthlyTipView.batValueView.amountLabel.text = "\(Int(self.recurringTipAmount))"
```

Comment:
This is set way later down the line already, is this needed? Wont it always be 0